### PR TITLE
#109 improve accessibility

### PR DESCRIPTION
**Pull request by Pdominguez15**

### CODE REVIEW

File: front/src/common-app/components/table-player.component.tsx [hunk 1]
```diff
@@ -17,71 +18,88 @@ interface Props {
...
+        <Typography
+          className={classes.subtitle}
+          variant="h6"
+          component={headingLevel}
+          id="headTable"
+        >
+          Players and votes:
+        </Typography>
+        <TableContainer className={classes.table}>
+          <Table aria-label="customized table" ref={ref} tabIndex={0}>
```
Comment (Nasdan) [inline front/src/common-app/components/table-player.component.tsx:36]: Replae `aria-label="customized table"` by `aria-labelledby="headTable"`

File: front/src/common-app/components/table-player.component.tsx [hunk 2]
```diff
@@ -17,71 +18,88 @@ interface Props {
...
+            <TableHead>
+              <TableRow>
+                <TableCell
+                  className={cx(classes.head, classes.cell)}
+                  aria-labelledby="headTable"
```
Comment (Nasdan) [inline front/src/common-app/components/table-player.component.tsx:41]: Remove `aria-labelledby="headTable"` from here

File: front/src/common-app/components/table-player.component.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import * as classes from './table-player.styles';
 import Typography from '@material-ui/core/Typography';
 import CheckIcon from '@material-ui/icons/Check';
 import CloseIcon from '@material-ui/icons/Close';
+import CloudCircleIcon from '@material-ui/icons/CloudCircle';
```
Comment (Nasdan) [inline front/src/common-app/components/table-player.component.tsx:7]: Replace by `import PendingVoteIcon from '@material-ui/icons/ViewCarousel';`

File: front/src/pods/player/player.component.tsx [hunk 4]
```diff
@@ -146,7 +146,7 @@ const ShowVotingResultsComponent: React.FC<ShowVotingResultsProps> = props => {
           />
         </div>
         <div className={classes.containerComponent}>
-          <h2 className={classes.title}>Wait for next story</h2>
```
Comment (Nasdan) [inline front/src/pods/player/player.component.tsx:149]: Remove h2 component with text `Show voting results` line [140](https://github.com/Lemoncode/planning-poker-example/pull/112/files#diff-cfde6183bcd22952f17e0bd765c1f25113b7b9e4957c356b83336722605acf20R140) as comment on point 7:

`7. Cuando el estudiante ve los resultados, quitar el texto Show voting results (comenta que no aporta nada), dejar tal y como está el h3 de "Players connected"`

File: front/src/pods/master/master.component.tsx [hunk 5]
```diff
@@ -98,6 +98,7 @@ const CreatingStoryComponent: React.FC<CreatingStoryProps> = props => {
   return (
     <>
       <div className={cx(classes.containerComponent, classes.leftContainer2)}>
+        {' '}
```
Comment (Nasdan) [inline front/src/pods/master/master.component.tsx:101]: Remote this

File: front/src/common-app/components/vote-options.component.tsx [hunk 6]
```diff
@@ -33,13 +35,13 @@ export const VoteOptionsComponent: React.FC<Props> = props => {
     voteCollection.length === 1 ? classes.contanierLabelShowVote : '';
 
   return (
-    <div className={classes.container}>
+    <div className={classes.container} ref={elementFocus} tabIndex={0}>
```
Comment (Nasdan) [inline front/src/common-app/components/vote-options.component.tsx:38]: Move ref and tabIndex to `Typography h2 Your Vote...` component

File: front/src/common-app/components/vote-options.styles.ts [hunk 7]
```diff
@@ -40,6 +40,7 @@ export const buttonContainer = css`
 export const container = css`
   display: flex;
   flex-flow: column;
+  outline: none !important;
```
Comment (Nasdan) [inline front/src/common-app/components/vote-options.styles.ts:43]: Use this style on `subtitle` class, Should be not necessary the `!important`

File: front/src/common-app/components/vote-options.component.tsx [hunk 8]
```diff
@@ -25,6 +26,7 @@ export const VoteOptionsComponent: React.FC<Props> = props => {
   const { showMessage } = useSnackbarContext();
 
   const onLocalVoteChosen = voteActive => {
+    elementFocus.current.focus();
```
Comment (Nasdan) [inline front/src/common-app/components/vote-options.component.tsx:29]: Remove this line and use: 

```
  React.useEffect(() => {
    if(votedStatus) {
      elementFocus.current.focus();
    }
  }, [votedStatus])
```

File: front/src/common-app/components/vote-options.component.tsx [hunk 9]
```diff
@@ -16,6 +16,7 @@ export const VoteOptionsComponent: React.FC<Props> = props => {
   const { onVoteChosen, votedStatus, buttonFinishVoting } = props;
   const [voteChosen, setVoteChosen] = React.useState('');
   const [voteActive, setVoteActive] = React.useState('');
+  const elementFocus = React.useRef<HTMLDivElement>(null);
```
Comment (Nasdan) [inline front/src/common-app/components/vote-options.component.tsx:19]: rename to `yourVoteElement` and replace `HTMLDivElement` by the correct one

File: front/src/pods/master/master.component.tsx [hunk 10]
```diff
@@ -165,6 +174,14 @@ interface ShowVotingResultsProps {
 
 const ShowVotingResultsComponent: React.FC<ShowVotingResultsProps> = props => {
   const { onMoveToNextStory, playerVotingStatus, title } = props;
+  const refTabla = React.useRef<HTMLTableElement>(null);
```
Comment (Nasdan) [inline front/src/pods/master/master.component.tsx:177]: tableRef

**Comment by Nasdan** [comment]
Nice work @Pdominguez15, some minor comments. Thank you

File: front/src/common-app/components/table-player.component.tsx (hunk 1 above)
Comment (Pdominguez15) [reply front/src/common-app/components/table-player.component.tsx:36]: Fix

File: front/src/common-app/components/table-player.component.tsx (hunk 2 above)
Comment (Pdominguez15) [reply front/src/common-app/components/table-player.component.tsx:41]: Fix

File: front/src/common-app/components/table-player.component.tsx (hunk 3 above)
Comment (Pdominguez15) [reply front/src/common-app/components/table-player.component.tsx:7]: Fix

File: front/src/pods/player/player.component.tsx (hunk 4 above)
Comment (Pdominguez15) [reply front/src/pods/player/player.component.tsx:149]: This is a mistake? This is not a Show voting results....

It's in the same component but up in the code.

I fix that code...

`
<div className={classes.rightContainer}>
        <div className={classes.containerComponent}>
//I will delete this H2
          <h2 className={classes.title}>Show voting results</h2>
        </div>
        <div className={classes.containerComponent}>
          <TablePlayerComponent
            playersCollection={voteCollectionResult}
            headingLevel="h3"
          />
        </div>
        <div className={classes.containerComponent}>
          <p className={classes.title}>Wait for next story</p>
        </div>
      </div>

`

File: front/src/pods/master/master.component.tsx (hunk 5 above)
Comment (Pdominguez15) [reply front/src/pods/master/master.component.tsx:101]: Fix

File: front/src/common-app/components/vote-options.component.tsx (hunk 6 above)
Comment (Pdominguez15) [reply front/src/common-app/components/vote-options.component.tsx:38]: Fix

File: front/src/common-app/components/vote-options.styles.ts (hunk 7 above)
Comment (Pdominguez15) [reply front/src/common-app/components/vote-options.styles.ts:43]: Fix

File: front/src/common-app/components/vote-options.component.tsx (hunk 8 above)
Comment (Pdominguez15) [reply front/src/common-app/components/vote-options.component.tsx:29]: Fix

File: front/src/common-app/components/vote-options.component.tsx (hunk 9 above)
Comment (Pdominguez15) [reply front/src/common-app/components/vote-options.component.tsx:19]: Fix

File: front/src/pods/master/master.component.tsx (hunk 10 above)
Comment (Pdominguez15) [reply front/src/pods/master/master.component.tsx:177]: Fix

File: front/src/pods/player/player.component.tsx (hunk 4 above)
Comment (Nasdan) [reply front/src/pods/player/player.component.tsx:149]: You are right, it above the line code in line 140, but Github doesn't allow write comments in collapsed lines. Thank you for the update

**Comment by Nasdan** [comment]
Thank you for updates @Pdominguez15, I will merge it

**Pull request merged**